library/tasks/ZarrUtils.wdl

```diff
@@ -93,6 +96,7 @@ task OptimusZarrConversion {
     set -euo pipefail
 
     python3 /tools/create_zarr_optimus.py \
+       --annotation_file ${annotation_file}\
```

Comment:
This works OK since the lines are indented but it looks odd to me. Can you add an extra space before the `\`s in this command block?
```suggestion
       --annotation_file ${annotation_file} \
```